content-src/components/MoreRecommendations/MoreRecommendations.jsx

```diff
@@ -7,7 +6,7 @@ export class MoreRecommendations extends React.PureComponent {
     if (read_more_endpoint) {
       return (
         <a className="more-recommendations" href={read_more_endpoint}>
-          <FormattedMessage id="pocket_more_reccommendations" />
+          <span data-l10n-id="newtab-pocket-more-recommendations" />
```

Comment:
Can't do standalone anchor or need to ignore (eslint(jsx-a11y/anchor-has-content) - Anchor nodes can't be empty and need screen-reader accessible content)

Reply:
@Mardak is it ok to specify default text to make eslint happy e.g. <a className="more-recommendations" href={read_more_endpoint} data-l10n-id="newtab-pocket-more-recommendations">More recommendations</a>

Reply:
We can add it to https://github.com/mozilla/activity-stream/blob/master/.eslintrc.js#L48-L55

Reply:
We can add it to https://github.com/mozilla/activity-stream/blob/1139bfc682ff544697ba5289a58b7623f3bc72d0/.eslintrc.js#L48-L55